charts/zenko/values.yaml

```diff
@@ -46,3 +46,8 @@ prometheus:
 mongodb-replicaset:
   replicaSet: rs0
   replicas: 5
+  image:
```

Comment:
I assume this has been tested. The approach is right, so if it works LGTM.

Reply:
Yes it does. I could successfully register my instance into orbit and create a bucket and

Reply:
So far 3.6.2 is the latest stable so lets go for it